community/shell/src/main/resources/META-INF/services/org.neo4j.shell.App

```diff
@@ -34,14 +34,9 @@ org.neo4j.shell.kernel.apps.Rmnode
 org.neo4j.shell.kernel.apps.Rollback
 org.neo4j.shell.kernel.apps.Set
 org.neo4j.shell.kernel.apps.Trav
-org.neo4j.shell.kernel.apps.Index
 org.neo4j.shell.kernel.apps.Gsh
 org.neo4j.shell.kernel.apps.Jsh
 org.neo4j.shell.kernel.apps.Dbinfo
 org.neo4j.shell.kernel.apps.IndexProviderShellApp
 org.neo4j.shell.kernel.apps.Schema
-org.neo4j.shell.apps.ha.Hainfo
-org.neo4j.shell.apps.ha.Pullupdates
```

Comment:
I don't think it is available in here, it is an enterprise thing. It is added here instead https://github.com/neo4j/neo4j/blob/3.0/enterprise/ha/src/main/resources/META-INF/services/org.neo4j.shell.App#L1